app/assets/stylesheets/components/_footer.scss

```diff
@@ -0,0 +1,49 @@
+footer {
+  display: flex;
+  align-items: center;
+  background-color: rgb(232, 230, 230);
+  width: 95%;
+  height: 200px;
+  border-radius: 15px;
+  margin: 0 auto;
+  margin-bottom: 15px;
+  padding: 15px;
+
+  .icon{
+    margin: 0 auto;
+    color: black;
+
+    &:hover {
+      color: #55679b;
+    }
+  }
+
+  .description {
+    margin: 0 auto;
+
+    li{
+      list-style: none;
+      text-align: center;
+
+      a {
+        font-size: 14px;
+        text-decoration: none;
+        text-align: center;
+        color: #8f8f8f;
+
+        &:hover {
+          color: #55679b;
+        }
+      }
```

Comment:
can you use $main-grey and $gold for the colors instead to fit the current design ?